src/io/flutter/view/InspectorPerfTab.java

```diff
@@ -131,13 +138,17 @@ private void buildUI() {
   private void setTrackRebuildWidgets(boolean selected) {
     final FlutterWidgetPerfManager widgetPerfManager = FlutterWidgetPerfManager.getInstance(app.getProject());
     widgetPerfManager.setTrackRebuildWidgets(selected);
+    // Send analytics.
+    sendAnalyticEvent(selected, "TrackRebuildWidgets");
```

Comment:
Actually, I think we should move the system of record for the rebuild/repaint prefs to the FlutterSettings class, and then track analytics from there. That will mean that we can better know how well used the setting is (esp. given that its currently opt-in).

(https://github.com/flutter/flutter-intellij/pull/2832 is related here)

Reply:
Great idea.  Now that you've done more consolidation in #2832, especially.

I'll update this once you land it.

Thanks!

Reply:
I think TrackRebuildWidgets is more like other boolean ext.flutter.* debugging extensions like 
RepaintRainbows, RepaintBaselines, and ShowPerfOverlays
so the better place to track it is in VMServiceManager which is being used as the single source of truth for those flags.
Another distinction is setting in FlutterSettings should persist across sessions but TrackRebuildWidgets shoudn't any more than DebugPaintRainbows should.

Reply:
@jacob314: are we tracking analytics for those service extension calls?  I didn't think we were...

Reply:
> I think TrackRebuildWidgets is more like other boolean ext.flutter.* debugging extensions like
RepaintRainbows, RepaintBaselines, and ShowPerfOverlays

👍 makes sense

Reply:
We should start tracking the analytics for them VMServiceManager as that is a good centralized place to track them. With the CL @kenzieschmoll is working on, the VMServiceManager becomes the centralized place all these extensions set and automatically reapplied on a hot restart

Reply:
Sounds good.  I can follow-up with another round of analytics once that lands.

In the meantime, do we agree that this is we want to track?

That is, an `enableFoo` and `disableFoo` for each event type `Foo`?  Or is `enable` sufficient?